code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -836,6 +836,9 @@
 	if(core)
 		core.resources = src.resources
 		src.pay_resources(src.resources) // just in case any weirdness happens let's pre-empt the dupe bug
+
+	if(absorber.item)
+		absorber.drop(absorber.item)
```

Comment:
maybe call `src.u_equip` here instead of manually calling `dropped` in the limb code? It should handle critter limbs fine

Reply:
It's not a limb, is the thing

Reply:
I said this in discord, but it's an equipment slot which should also be handled fine